tools/server/server-models.cpp

```diff
@@ -968,7 +1005,10 @@ server_http_proxy::server_http_proxy(
     if (pipe->read(header, should_stop)) {
         SRV_DBG("%s", "received response headers\n");
         this->status  = header.status;
-        this->headers = header.headers;
+        this->headers = std::move(header.headers);
+        if (!header.content_type.empty()) {
+            this->content_type = std::move(header.content_type);
```

Comment:
If we std::move here, I prefer restrict the scope of `header` to avoid someone (in the future) accidentally use `header` after this point